packages/nodes-base/nodes/ErrorTrigger/ErrorTrigger.node.ts

```diff
@@ -54,7 +54,7 @@ export class ErrorTrigger implements INodeType {
 			items[0].json = {
 				execution: {
 					id,
-					url: `${urlParts.join('/')}/${id}`,
+					url: `${urlParts.join('/')}/workflow/1/${id}`,
```

Comment:
the hardcoded '1' has a smell... 🤔

Reply:
This is for the manual execution for testing out the node output. It's not the actual behaviour of the node.

Reply:
ah, ok. Thanks!